pii_team_sei.py

```diff
@@ -34,4 +34,5 @@ def find_email(text) -> list:
 def find_instagram_handle(text) -> list:
     """Finds all occurrences of an instagram handle in a text string"""
     # match an instagram handle
-    return []
\ No newline at end of file
+    return []
+    return re.findall(r'(?<!\S)@[\w\d.]{1,30}', text)
```

Comment:
The first return [] will return an empty list and the second return is never executed.